guides/common/modules/proc_installing-the-dhcp-infoblox-module.adoc

```diff
@@ -29,9 +29,6 @@ You cannot use the `host` option without creating conflicts and, for example, be
 --foreman-proxy-plugin-dhcp-infoblox-record-type fixedaddress \
 --foreman-proxy-plugin-dhcp-infoblox-username admin
```

Comment:
I think this (and password) need to be
```suggestion
--foreman-proxy-plugin-dhcp-infoblox-username _admin_
```

Also, you can drop `--foreman-proxy-dhcp-managed` since it's ignored when the provider is set to infoblox.